fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -503,18 +532,18 @@ struct TenantManagementWorkload : TestWorkload {
 					}
 
 					// Check the state of the first created tenant
-					Optional<TenantMapEntry> resultEntry =
-					    wait(self->tryGetTenant(tenantsToCreate.begin()->first, operationType));
-
-					if (resultEntry.present()) {
-						if (resultEntry.get().tenantState == TenantState::READY) {
-							// The tenant now exists, so we will retry and expect the creation to react accordingly
-							alreadyExists = true;
-						} else {
-							// Only a metacluster tenant creation can end up in a partially created state
-							// We should be able to retry and pick up where we left off
-							ASSERT(operationType == OperationType::METACLUSTER);
-							ASSERT(resultEntry.get().tenantState == TenantState::REGISTERING);
+					if (operationType == OperationType::METACLUSTER) {
```

Comment:
I think these checks need to be run regardless of the operation type, but for a non-metacluster we can just observe the presence or absence of the tenant.